app.js

```diff
@@ -0,0 +1,190 @@
+var app = require('koa')();
+var router = require('koa-router')();
+var koaBody   = require('koa-body')();
+var fs = require('fs');
+var staticServe = require('koa-static');
+
+var dataStorage = './dataStorage/';
+
+
+// REST
+router.get('/notes', function*(next) {
+
+  yield new Promise((resolve, reject) => {
```

Comment:
better to wrap all the needed fs calls with promises, then use those in your routes because 1) you don't need to wrap same function multiple times, and 2) makes your router code cleaner, especially with a framework like koa.